src/Vixen.Common/Controls/TimeLineControl/Grid_Mouse.cs

```diff
@@ -967,7 +967,7 @@ private void MouseMove_HResizing(Point gridLocation, Point delta)
 					if (AltPressed && adjoiningElements.Count() == 0)
 					{
 						// Get the Elements that are preceding the Selected Element
-						List<Element> priorElements = Rows.First().GetPriorsOfElement(SelectedElements.First());
+						List<Element> priorElements = ActiveRow.GetPriorsOfElement(SelectedElements.First());

```

Comment:
ActiveRow may not always be the same as the effect you are trying to adjust. Example. Drag a new effect to a row or an effect from any row to a new row. That new effect will be selected, and you can move / resize it, but the active row will not coincide. So probably the safest solution here is to capture SelectedElements.First() and then use the Row of that element which should be the row you want.

Reply:
Changing the asignment of ActiveRow may have unforseen ramifications. I instead changed to use your recommendation.